022-quarkus-properties-config-all/src/test/java/quarkus/qe/GreetingResourceTest.java

```diff
@@ -0,0 +1,69 @@
+package quarkus.qe;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+public class GreetingResourceTest {
+
+    @Test
+    public void testEnvFile() {
+        given()
+          .when().get("/hello")
+          .then()
+             .statusCode(200)
+             .body(is("Welcome message from .env file"));
+    }
+
+    @Test
+    public void testProtagonistEndpoint() {
```

Comment:
Indicate what is the source of the configuration (application properties, yaml, resources-config.json etc) in the test name
Do not use the endpoint name, that doesn't hint the problematic area if the test fails

Reply:
Good point. Changed